Prevent collision with UITableView

PeterGumball opened the pull request:
Limit the width for the PanGestureRecognizer to prevent collision with
UITableView (swipe to delete)

Commit:
Prevent collision with UITableView

Limit the width for the PanGestureRecognizer to prevent collision with
UITableView (swipe to delete)

alexkrzyzanowski commented:
Thank you for pull request @PeterGumball . I'll check it a bit later and try to cover code with automated tests.

Commit:
Prevent collision with UITableView

Add a offset for the height of PanGestureRecognizer in order to handle
a tab on NavigationBar.

PeterGumball commented:
Hey @alexkrzyzanowski. 

 Sorry for this commit. Small bug fixing. ;)

alexkrzyzanowski commented:
Actually, it's still a bit incorrect, because height goes out of the screen and the next line should be improved in a manner like:

```
let panView = UIView(frame: CGRectMake(0, CGFloat(heightOffsetForPanGestureRecognizer), CGFloat(widthForPanGestureRecognizer), self.view.frame.size.height - heightOffsetForPanGestureRecognizer))
```

Anyway I'll merge it because it is a solution. But not the best one. Today evening I'll try to recognize right-to-left swipe gesture to pass it to `UITableView`. I don't know if it is possible or not, but at least I can try.

PeterGumball commented:
Oh yes, i forgot.

I now it's not the best solution. ;)

alexkrzyzanowski commented:
I've already merged the PR :). I'll try to implement the best solution in a feature branch. If you want to be notified about progress on this issue, you can subscribe to issue https://github.com/alexkrzyzanowski/SidebarOverlay/issues/19.

PeterGumball commented:
Ok, thanks for the info.
